cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -223,7 +223,10 @@ metadata:
   labels:
     k8s-app: kube-dns
 spec:
-  replicas: 2
+  # replicas: not specified here:
+  # 1. In order to make Addon Manager do not reconcile this replicas parameter.
```

Comment:
does kubeadm have an "addon manager" ? @neolit123

Reply:
no, it does not have one, per se.
it has "phases" that manage addons.

Reply:
the comment can be:
`# Default replica count is 1`

Reply:
right, that's what I thought re: phases. 
I think the rest of the details in this comment make more sense for kube-up and less sense for kubeadm (presuming this is referring to the "Addon manager" in cluster/)